src/main/java/Duke.java

```diff
@@ -1,10 +1,171 @@
+        int taskDescriptionStartIndex;
+        int taskDescriptionEndIndex;
+        
+        //find start of task description
+        taskDescriptionStartIndex = userInput.indexOf(" ");
+        taskDescriptionEndIndex = userInput.indexOf("/by");
+        //assign task's information
+        description = userInput.substring(taskDescriptionStartIndex + 1, taskDescriptionEndIndex).trim();
+        String by = userInput.substring(taskDescriptionEndIndex + 3).trim();//+3 to exclude "/by"
+        taskType = TaskType.DEADLINE;
+        //create and assign information to new deadline
+        Deadline deadline = new Deadline(description,taskType,by);
+        tasks[listSize++] = deadline;
+    }
+    
+    //add todo to list
+    private static void addTodo(String userInput) {
+        TaskType taskType;
+        String description;
+        
+        //start index for description
+        int taskDescriptionStartIndex;
+        
+        //find start of task description
+        taskDescriptionStartIndex = userInput.indexOf(" ");
+        //assign task's information
+        description = userInput.substring(taskDescriptionStartIndex + 1);
+        taskType = TaskType.TODO;
+        //create and assign information to new ToDo
+        ToDo toDo = new ToDo(description,taskType);
+        tasks[listSize++] = toDo;
+    }
+    
+    //display current list
+    public static void displayList(){
+        System.out.print(
+                "____________________________________________________________\n"+
+                " Here are the tasks in your list:\n"
+        );
+        //print out all list's items
+        for (int i = 0;i < listSize;i++){
+            tasks[i].displayListPerTask(i);
+        }
+        System.out.print(
+                "____________________________________________________________\n"
+        );
+    }
+    
+    //handle user input commands
+    private static void handleCommand() {
+        //activate scanner
+        Scanner input = new Scanner(System.in);
+        String userInput; //user input
+        
+        //scan user's input
+        userInput = input.nextLine();
+        
+        //execute command based on user's input until "bye" is entered
+        while(!userInput.equals("bye")){
+            if(userInput.equals("list")){
+                //display list
+                displayList();
+            } else if(userInput.startsWith("done")){
+                //mark the given activity as done
+                markDone(userInput);
+            } else {
+                //add user input to list
+                addList(userInput);
+            }
+            //receive next user command
+            userInput = input.nextLine();
+        }
+    }
+    
+    //mark an activity as done
+    private static void markDone(String userInput) {
+        //take word after "done" as task index
+        String taskIndexString = userInput.split(" ")[1];
+        Integer taskIndex = Integer.parseInt(taskIndexString);
```

Comment:
Hello! Perhaps, you can add comment on this one for signifying a conversion (for example: // convert taskIndexString into an integer)